go/client/cmd_signup.go

```diff
@@ -205,17 +205,13 @@ func (s *CmdSignup) prompt() (err error) {
 		s.G().Log.Debug("| Prompter failed\n")
 		return
 	}
-	arg := keybase1.GetNewPassphraseArg{
-		TerminalPrompt: "Pick a strong passphrase",
-		PinentryDesc:   "Pick a strong passphrase (12+ characters)",
-		PinentryPrompt: "Passphrase",
-		UseSecretStore: libkb.HasSecretStore(),
-	}
 
 	f := s.fields.passphraseRetry
 	if f.Disabled || libkb.IsYes(f.GetValue()) {
+		// note that since this is running in the client, it is not
+		// possible to use a delegated SecretUI.
 		var res keybase1.GetPassphraseRes
-		res, err = s.G().UI.GetSecretUI().GetNewPassphrase(arg)
+		res, err = libkb.GetSignupPassphrase(s.G().UI.GetSecretUI())
```

Comment:
note that this code isn't only reachable from standalone mode. it's also reachable in client/server mode where the client is a CLI.